api/service/manager.go

```diff
@@ -82,6 +85,11 @@ func (m *Manager) GetServices() []Service {
 	return m.services
 }
 
+// GetService get the specified service
+func (m *Manager) GetService(t Type) Service {
+	return m.serviceMap[t]
```

Comment:
any lock needed?

Reply:
I don't think we will need lock now since the serviceMap is fixed.